build.gradle

```diff
@@ -281,9 +281,9 @@ configure(subprojects) {
         }
     }
 
-    checkstyle {
-        toolVersion '8.19'
-        configFile = file("${rootProject.projectDir}/develop/checkstyle-rules.xml")
+    checkstyle{
+        toolVersion=8.41
```

Comment:
Maybe an additional whitespace here? ;) Otherwise the results look good and I detected no further problems.